src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Objects/Attributes.jl

```diff
@@ -583,3 +583,44 @@ function relative_cotangent_module(X::AbsProjectiveScheme{<:Ring, <:MPolyQuoRing
   return cokernel(psi)
 end
 
+@doc raw"""
+    genus(X::AbsProjectiveScheme{<:Field}; algorithm::Symbol=:nor)
```

Comment:
```suggestion
    genus(X::AbsProjectiveScheme{<:Field}; algorithm::Symbol=:normalization)
```
inconsistent with signature